src/main/java/uk/gov/hmcts/darts/common/util/FileContentChecksum.java

```diff
@@ -40,18 +36,9 @@ public String calculate(InputStream inputStream) {
         return encodeToString(md5(inputStream));
     }
 
-    /**
-     * Calculates the digest when the source data has already been consumed.
-     */
-    public String calculate(DigestInputStream digestInputStream) throws IOException {
-        return encodeToString(digestInputStream.getMessageDigest().digest());
-    }
-
     @SneakyThrows
     public String calculate(Path filePath) {
-        ByteSource byteSource = com.google.common.io.Files.asByteSource(filePath.toFile());
-        HashCode hc = byteSource.hash(Hashing.md5());
-        return encodeToString(hc.asBytes());
+        return calculate(new FileInputStream(filePath.toFile()));
```

Comment:
Author note: @mario-paniccia I've taken opportunity here to rework this method to use the same underlying `org.apache.commons.codec.digest.DigestUtils.md5()` method as the others in this class.

Reply:
nice one cheers @chrisbellingham-hmcts